pkg/generator/generator.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	DummyIgnition      bool   `envconfig:"DUMMY_IGNITION"`
 	InstallInvoker     string `envconfig:"INSTALL_INVOKER" default:"assisted-installer"`
 	ServiceBaseURL     string `envconfig:"SERVICE_BASE_URL"`
+	InstallerCacheSize int    `envconfig:"INSTALLER_CACHE_SIZE" default:"5"`
```

Comment:
let's add it in ``openshift/template.yaml`` so we can control it for the different environments (in case we'll need to)

Reply:
Added